frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1103,10 +1103,18 @@ function ReaderRolling:updateBatteryState()
         local powerd = Device:getPowerDevice()
         -- -1 is CR_BATTERY_STATE_CHARGING @ crengine/crengine/include/lvdocview.h
         local state = powerd:isCharging() and -1 or powerd:getCapacity()
+        if powerd.device:hasAuxBattery() and powerd:isAuxBatteryConnected() and
+            not powerd:isAuxCharging() then
+            -- If aux_battery not charging, but present -> don't show '[ + ]' in header
+            -- but show the average (as both battery have the same maximum capacity).
+            state = math.floor((powerd:getCapacity() + powerd:getAuxCapacity() + 1) / 2)
```

Comment:
I'm... not a fan of that ;).

It's extremely irregular and inconsistent.

I'd much rather we stick to the ereader battery.

Reply:
It is not possible to show both batteries separately in crengine's status line. But it is possible to show more than 100% ;)

So what about
```
state = powerd:getCapacity() + powerd:getAuxCapacity()
```
As the cover is always charging the internal battery, 145% would mean 100% internal battery and 45% cover capacity.
(And yes, for now this is only for the Sage+Power-cover)

Reply:
But then you'd show 100% when it's 50%+50%. Seems a bit deceptive?

Reply:
Yes, but this is only a transient state, as the cover starts immediately chargeing the reader. After some time the charge would be 100+0.
And anyway the status line shows the remaining total capacity.

Reply:
When a cover is connected, the cover always try to charge the reader to 100%.

Reply:
Had I such a cover, I would be okay with seeing just the sum (0-200%).

Reply:
[0,200] sounds good to me, yeah ;).

The charging behavior is to charge the reader first, so unbalanced gauges should be fairly rare and/or transient.

(Also, you usually don't read while inside the cover, because ahahaha, it's almost as heavy as a laptop ;p).